books/free-programming-books-fr.md

```diff
@@ -323,6 +323,7 @@
 ### SQL
 
 * [Cours complet pour apprendre les différents types de bases de données et le langage SQL](https://sgbd.developpez.com/tutoriels/cours-complet-bdd-sql/) - Jacques Le Maitre
+* [Cours de SQL base du langage SQL et des bases de données](https://sql.sh) - (en ligne, PDF, EPUB)
```

Comment:
```suggestion
* [Cours de SQL base du langage SQL et des bases de données](https://sql.sh) - Tony Archambeau
```
Please add author names where appropriate.
don't need list of filetypes if everything is accessible from the linked page

Reply:
it's done